Gruntfile.js

```diff
@@ -335,7 +335,7 @@ module.exports = function (grunt) {
                     // The need for this should go away once the version of phantomjs gets upgraded to 2.0
                     'test/polyfills.js',
 
-                    'src/thirdparty/jquery-2.1.3.min.js',
+                    'src/thirdparty/jquery-3.2.1.min.js',
```

Comment:
Hmm upgrade to 3.2.1 seems dangerous here, can we do it in a separate PR?

Reply:
@petetnt Ok. I'll try it
Thank you!

Reply:
#13854
@petetnt There is a separate PR :)